atom/browser/api/atom_api_app.cc

```diff
@@ -47,7 +47,6 @@
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/client_certificate_delegate.h"
 #include "content/public/browser/gpu_data_manager.h"
-#include "content/public/browser/memory_pressure_controller.h"
```

Comment:
why this change? If `MemoryPressureListener` moved we still need an include for it